WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewController.swift

```diff
@@ -29,7 +29,7 @@ private extension InPersonPaymentsMenuViewController {
         rows = [
             .orderCardReader,
             .manageCardReader,
-            .bbposChipper2XBTManual
+            .stripeM2Manual
```

Comment:
I was a little surprised that we are not going to continue to show the user manual for the installed base of chippers, but I will assume this is correct.

Reply:
Also, while we are changing this, maybe it would be best to make it more generic, i.e. `.readerManual` and then we just have to update the link in the future when Stripe releases an M3 :)

Reply:
We discussed this earlier today and we are going to display the links to both manuals. I'll push an update later today